src/main/java/de/zalando/aruha/nakadi/controller/EventTypeController.java

```diff
@@ -0,0 +1,132 @@
+package de.zalando.aruha.nakadi.controller;
+
+import de.zalando.aruha.nakadi.NakadiException;
+import de.zalando.aruha.nakadi.domain.EventType;
+import de.zalando.aruha.nakadi.problem.DuplicatedEventTypeNameProblem;
+import de.zalando.aruha.nakadi.problem.ValidationProblem;
+import de.zalando.aruha.nakadi.repository.DuplicatedEventTypeNameException;
+import de.zalando.aruha.nakadi.repository.EventTypeRepository;
+import de.zalando.aruha.nakadi.repository.NoSuchEventTypeException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.Errors;
+import org.springframework.web.bind.annotation.*;
+import org.zalando.problem.MoreStatus;
+import org.zalando.problem.Problem;
+
+import javax.validation.Valid;
+import javax.ws.rs.core.Response;
+import java.net.URI;
+import java.util.List;
+
+import static org.springframework.http.ResponseEntity.status;
+
+@RestController
+@RequestMapping(value = "/event_types")
+public class EventTypeController {
+
+    final private EventTypeRepository repository;
+
+    @Autowired
+    public EventTypeController(EventTypeRepository repository) {
+        this.repository = repository;
+    }
+
+    @RequestMapping(method = RequestMethod.GET)
+    public ResponseEntity<?> list() {
+        List<EventType> eventTypes = repository.list();
+
+        return status(HttpStatus.OK).body(eventTypes);
+    }
+
+    @RequestMapping(method = RequestMethod.POST)
+    public ResponseEntity<?> createEventType(@Valid @RequestBody final EventType eventType, final Errors errors) throws Exception {
+        if (errors.hasErrors()) {
+            final Problem problem = new ValidationProblem(errors);
+
+            return status(HttpStatus.UNPROCESSABLE_ENTITY).body(problem);
+        } else {
+            return persist(eventType);
+        }
+    }
+
+    @RequestMapping(value = "/{name}", method = RequestMethod.PUT)
```

Comment:
I think these references are remnants of old definitions. Topic field has been completely removed in here https://github.com/zalando/nakadi/commit/16c0c903c0ffe4c593127c0f3d93f1893d9b1fbb